include/mbgl/gl/location_component_layer.hpp

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#ifndef GL_GLEXT_PROTOTYPES
+#define GL_GLEXT_PROTOTYPES
+#endif
+
+#include <mapbox/cheap_ruler.hpp>
+#include <mapbox/geometry.hpp>
+#include <mbgl/gl/uniform.hpp>
+#include <mbgl/platform/gl_functions.hpp>
+#include <mbgl/style/style.hpp>
+#include <mbgl/util/convert.hpp>
+#include <mbgl/util/geo.hpp>
+#include <mbgl/util/mat4.hpp>
+#include <mbgl/util/projection.hpp>
+
+#include <array>
+#include <mbgl/style/expression/formatted.hpp>
+#include <mbgl/style/filter.hpp>
+#include <mbgl/style/layer.hpp>
+#include <mbgl/style/property_value.hpp>
+#include <mbgl/util/color.hpp>
+
+using namespace mbgl;
+using namespace mbgl::style;
```

Comment:
`using namespace` cannot be used in a header

Reply:
done